<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>blogsView</class>
 <widget class="QMainWindow" name="blogsView">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QScrollArea" name="scrollArea">
      <property name="widgetResizable">
       <bool>true</bool>
      </property>
      <widget class="QWidget" name="scrollAreaWidgetContents">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>780</width>
         <height>539</height>
        </rect>
       </property>
       <layout class="QGridLayout" name="gridLayout_2">
        <item row="0" column="0">
         <layout class="QVBoxLayout" name="blogsLayout"/>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>19</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuManageBlogEntires">
    <property name="title">
     <string>Blogs</string>
    </property>
    <addaction name="actionCreate_new_Blog_entry"/>
   </widget>
   <widget class="QMenu" name="menuExit">
    <property name="title">
     <string>App</string>
    </property>
    <addaction name="actionExit"/>
   </widget>
   <addaction name="menuManageBlogEntires"/>
   <addaction name="menuExit"/>
  </widget>
  <action name="actionCreate_new_Blog_entry">
   <property name="text">
    <string>Create blog entry</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
